sdk/file_sdk/UnitTests/common_ut/extended_property_ut.cpp

```diff
@@ -0,0 +1,48 @@
+#include "extended_property_ut.h"
+#include <extended_property.h>
+
+using std::string;
+
+ExtendedPropertyTests::ExtendedPropertyTests() {
+}
+
+void SetExtendedProperty(mip::ExtendedProperty& property, string key, string vendor, string value ) {
+  property.key = key;
+  property.vendor = vendor;
+  property.value = value;
+}
+
+void ExtendedPropertyTests::Equal_SameProperty_ReturnTrue() {
+  mip::ExtendedProperty property1;
+  SetExtendedProperty(property1, "key", "vendor", "value");

```

Comment:
>"key", "vendor", "value" [](start = 33, length = 24)

you can init this way:
mip::ExtendedProperty property1 { "key", "vendor", "value"};